test/join.jl

```diff
@@ -578,4 +578,54 @@ module TestJoin
         @test eltypes(join(l, r, on=:b, kind=:outer, makeunique=true)) ==
             [Union{Int, Missing}, CS, Union{Int, Missing}]
     end
+
+    @testset "test checks of merge key uniqueness" begin
+        @test_throws ArgumentError join(name, job, on=:ID,
+                                        validate=(false, true))
```

Comment:
Better put this on the previous line, it's well below 92 chars.

Reply:
Julia's CONTRIBUTING.md mentions 92 chars, that's usually the target we try to follow.

Reply:
wow that is so much more breathing room